components/DatasetsList/index.tsx

```diff
@@ -0,0 +1,135 @@
+    <li className="app-datasets-list__item">
+      <div className="app-datasets-list__item-top">
+        <a
+          className="app-datasets-list__item-title app-datasets-list__item-title--context govuk-link"
+          href={props.creator}
+        >
+          {props?.datasetTitle}
+        </a>
+        <p className="app-datasets-list__item-description">
+          {props?.shortDescription}
+        </p>
+      </div>
+      <div className="app-datasets-list__item-bottom">
+        <div className="app-datasets-list__item-bottom-publisher">
+          <Image
+            src="/assets/images/crest.png"
+            width={40}
+            height={34}
+            alt="Govuk Crest"
+          />
+          <h3 className="app-datasets-list__item-publisher-inner">
+            {props?.publisher}
+          </h3>
+        </div>
+        <ul
+          className="app-datasets-list__item-metadata"
+          style={{ textAlign: "right" }}
+        >
+          <li className="app-datasets-list__item-metadata-row">
+            <div>{props?.topic}</div>
+          </li>
+          <li className="app-datasets-list__item-metadata-row">
+            <time dateTime={props.modified?.value}>{props?.subTopic}</time>
+          </li>
+          <li className="app-datasets-list__item-metadata-row">
+            <time dateTime={props.modified?.value}>
+              Updated: {formatDate(props.modified)}
+            </time>
+          </li>
+        </ul>
+      </div>
+    </li>
+  );
+}
+
+function formatDate(date: string) {
+  const d = new Date(date);
+  const options: Intl.DateTimeFormatOptions = {
+    day: "numeric",
+    month: "long",
+    year: "numeric",
+  };
+  return d
+    .toLocaleDateString("en-GB", options)
+    .replace(",", "")
+    .replace(/( AM| PM)/g, (m) => m.trim().toLowerCase());
+}
+
+export default function DatasetsList({
+  items,
+  page = 1,
+  searchParams,
+}: {
+  items: any;
+  page: number;
+  searchParams: URLSearchParams;
+}) {
+  const RESULTS_LENGTH = 20;
+  const sliceStart = page * RESULTS_LENGTH - RESULTS_LENGTH;
+  const sliceEnd = page * RESULTS_LENGTH;
+  const totalPages = Math.ceil(items.length / RESULTS_LENGTH);
+
+  // TODO this is a temp solution until we add in a correct sort by process
+  const sortByDate = (data: any[]) => {
+    return data.sort((a, b) => {
+      const dateA = new Date(a.modified).getTime();
+      const dateB = new Date(b.modified).getTime();
+      return dateB - dateA;
+    });
+  };
+
+  const sortedItems = sortByDate(items);
+
+  return (
+    <div className="govuk-grid-column-two-thirds-from-desktop">
+      <div className="app-datasets-header">
+        <h3 className="app-datasets-header__results">
+          {sortedItems.length} results
+        </h3>
+        <div
+          style={{
+            display: "flex",
+          }}
+        >
+          <MdRssFeed className="app-datasets-header__icon" />
+          <a className="app-datasets-header__link govuk-link" href={"#"}>
```

Comment:
Has this not been removed from the designs yet? Worth checking with Edd and Gareth if we want this in, I feel like I remember a conversation where it was agreed not to do this but it might not have got communicated

Reply:
Double checked with Edd and have removed it